### PR TITLE
mahout 0.11.0

### PR DESCRIPTION
**Pull request by stuwil**

### CODE REVIEW

File: Library/Formula/mahout.rb [hunk 1]
```diff
@@ -37,4 +37,15 @@ def caveats; <<-EOS.undent
       export JAVA_HOME=$(/usr/libexec/java_home)
     EOS
   end
+
+  test do
+    ENV["JAVA_HOME"] = "$(/usr/libexec/java_home)"
```
Comment (bfontaine) [inline Library/Formula/mahout.rb]: Use `` `/usr/libexec/java_home`.chomp`` here.

File: Library/Formula/mahout.rb [hunk 2]
```diff
@@ -37,4 +37,15 @@ def caveats; <<-EOS.undent
       export JAVA_HOME=$(/usr/libexec/java_home)
     EOS
   end
+
+  test do
+    ENV["JAVA_HOME"] = `/usr/libexec/java_home`.chomp
```
Comment (xu-cheng) [inline Library/Formula/mahout.rb]: We can set JAVA_HOME for users in install:
`bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env)`

File: Library/Formula/mahout.rb [hunk 3]
```diff
@@ -30,6 +30,7 @@ def install
     end
 
     bin.write_exec_script Dir["#{libexec}/bin/*"]
```
Comment (xu-cheng) [inline Library/Formula/mahout.rb]: You should delete this line. It's covered by line in below. 

Also can you remove `export JAVA_HOME` in caveats? thanks.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/11031332eadc8c38306ebee2ad6066f44742b3b1. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

**Pull request closed without merging**